src/app/process-editor/store/process-entities.state.ts

```diff
@@ -18,10 +18,17 @@
 import { EntityState, createEntityAdapter } from '@ngrx/entity';
 import { Process, ProcessContent } from 'ama-sdk';
 
+export interface SelectedProcessElement {
+    id: string;
+    type: string;
+    name?: string;
+}
+
 export interface ProcessEntitiesState extends EntityState<Process>  {
     loading: boolean;
     loaded: boolean;
-    entityContents: {[key: string]: ProcessContent};
+    selectedProcessContent: ProcessContent;
```

Comment:
These ones needs to be in the process-editor.state, it doesn't belong to the entity state.
This state file is supposed to store only information about the state, which is completely detached from the process-editor itself.
The same is applied for selectors, effects, actions, etc... Those are different responsibilities.
Like in the case of connector-editor.state.

Reply:
And by the way, where do we store the entityContents? :octocat: